x-pack/plugins/uptime/server/lib/adapters/telemetry/__tests__/kibana_telemetry_adapter.test.ts

```diff
@@ -6,6 +6,10 @@
 
 import { KibanaTelemetryAdapter } from '../kibana_telemetry_adapter';
 
+jest
+  .spyOn(KibanaTelemetryAdapter, 'countNoOfUniqueMonitorAndLocations')
+  .mockResolvedValue(undefined as any);
```

Comment:
@elastic/uptime This is necessary to get the tests passing because the error thrown by this method no longer is ignored. I'm assuming we want to write tests for this method, but I'm not familiar enough to fix it in this little path. Could someone from the uptime team make sure I'm not making things worse with this mock?

Reply:
I'm OK with merging this so long as tests are passing for now since it seems to affect CI stability. We'll make sure that the tests are fixed if need be.